public/app/plugins/datasource/prometheus/components/PromQueryEditor.tsx

```diff
@@ -24,11 +24,29 @@ const INTERVAL_FACTOR_OPTIONS: Array<SelectableValue<number>> = map([1, 2, 3, 4,
   label: '1/' + value,
 }));
 
+const STEP_OPTIONS: Array<SelectableValue<StepType>> = [
+  {
+    value: 'min',
+    label: 'minimum',
+  },
+  {
+    value: 'max',
+    label: 'maximum',
+  },
+  {
+    value: 'exact',
+    label: 'exact',
+  },
+];
+
+const DEFAULT_STEP_OPTION = STEP_OPTIONS[0];
```

Comment:
I think this shouldn't be the other way around. You specify the default step, and then use it in step options, as this could get broken if someone introduced a new step and added it to the STEP_OPTIONS array at the first place.